Gemfile

```diff
@@ -110,6 +111,7 @@ group :development, :test do
 end
 
 group :test do
+  gem 'phantomjs', :require => 'phantomjs/poltergeist'
```

Comment:
I tried adding this to my own local and I'm still seeing 6 failures. I am suspecting now it might be because I am running on Ubuntu. Are you on Mac?

Reply:
And what version do you see on your system? (`phantomjs --version`)

Reply:
I am on a mac, yes. This may have to do with whatever I ran into post-rebase, where I was hitting the capybara errors but they were outdated perhaps.

Here's what I see after I run `rspec`:

```Finished in 49.5 seconds (files took 7.22 seconds to load)
124 examples, 0 failures, 6 pending```

...also, WELP `zsh: command not found: phantomjs`

Reply:
Some more tests are still flaking occasionally for me:
```
rspec ./spec/features/pin_creation_spec.rb[1:2:2:1] # pin creation with js with surgeon and procedure initialized creates a new pin with data and images
rspec ./spec/features/pin_creation_spec.rb[1:2:2:2] # pin creation with js with surgeon and procedure initialized creates a pin, adding a new surgeon and procedure

# Randomized with seed 55443

rspec ./spec/features/pin_update_spec.rb[1:2:1] # pin updating with js updates the pin with one less photo and new info
# Randomized with seed 9152
```

The error messages for those may contain further clues, but I haven't gotten a chance to debug further.

```

  1) pin updating with js updates the pin with one less photo and new info
     Failure/Error: click_button "Submit Now"
     Capybara::Poltergeist::JavascriptError:
       One or more errors were raised in the Javascript code on the page. If you don't care about these errors, you can ignore them by setting js_errors: false in your Poltergeist configuration (see documentation for details).

       TypeError: null is not an object (evaluating 'e.takeRecords')
       TypeError: null is not an object (evaluating 'e.takeRecords')
           at https://pagead2.googlesyndication.com/pagead/js/r20210211/r20190131/rum.js:39 in H
     Shared Example Group: "pin updating" called from ./spec/features/pin_update_spec.rb:94
     # /home/corajr/.rvm/gems/ruby-2.3.3/gems/poltergeist-1.9.0/lib/capybara/poltergeist/browser.rb:351:in `command'
     # /home/corajr/.rvm/gems/ruby-2.3.3/gems/poltergeist-1.9.0/lib/capybara/poltergeist/browser.rb:180:in `click'
     # /home/corajr/.rvm/gems/ruby-2.3.3/gems/poltergeist-1.9.0/lib/capybara/poltergeist/node.rb:17:in `command'
     # /home/corajr/.rvm/gems/ruby-2.3.3/gems/poltergeist-1.9.0/lib/capybara/poltergeist/node.rb:131:in `click'
     # /home/corajr/.rvm/gems/ruby-2.3.3/gems/capybara-2.5.0/lib/capybara/node/element.rb:134:in `block in click'
     # /home/corajr/.rvm/gems/ruby-2.3.3/gems/capybara-2.5.0/lib/capybara/node/base.rb:84:in `synchronize'
     # /home/corajr/.rvm/gems/ruby-2.3.3/gems/capybara-2.5.0/lib/capybara/node/element.rb:134:in `click'
     # /home/corajr/.rvm/gems/ruby-2.3.3/gems/capybara-2.5.0/lib/capybara/node/actions.rb:39:in `click_button'
     # /home/corajr/.rvm/gems/ruby-2.3.3/gems/capybara-2.5.0/lib/capybara/session.rb:698:in `block (2 levels) in <class:Session>'
     # /home/corajr/.rvm/gems/ruby-2.3.3/gems/capybara-2.5.0/lib/capybara/dsl.rb:51:in `block (2 levels) in <module:DSL>'
     # ./spec/features/pin_update_spec.rb:36:in `block (3 levels) in <top (required)>'

Finished in 38.17 seconds (files took 5.02 seconds to load)
124 examples, 1 failure, 6 pending

```